src/transforms/lua/v2/mod.rs

```diff
@@ -73,6 +75,15 @@ pub struct LuaConfig {
     /// A list of timers which should be configured and executed periodically.
     #[serde(default)]
     timers: Vec<TimerConfig>,
+
+    /// When set to `single`, metric tag values will be exposed as single strings, the
+    /// same as they were before this config option. Tags with multiple values will show the last assigned value, and null values
+    /// will be ignored.
+    ///
+    /// When set to `full`, all metric tags will be exposed as arrays of either string or null
+    /// values.
+    #[serde(default)]
+    metric_tags_values: MetricTagsValues,
```

Comment:
I think we were going with `metric_tag_values` for this setting (here and in the docs below).